tests/unit/test_zigzag_config_file.py

```diff
@@ -101,6 +101,52 @@ def config_missing_project_id_key(tmpdir_factory):
     return config_path
 
 
+@pytest.fixture(scope='session')
+def module_hierarchy_two_nodes(tmpdir_factory):
+    """config with one value in a jinga template"""
```

Comment:
Description doesn't match intention of fixture.

Reply:
Updated